git_machete/client.py

```diff
@@ -2134,7 +2136,7 @@ def __derive_remote_and_github_org_and_repo(self,
                     'corresponds to GitHub (see `git remote -v` for details). \n'
                     'It is possible that you are using custom GitHub URL.\n'
                     'If that is the case, you can provide repository information explicitly, via these 3 git config keys: '
-                    '`machete.github.{remote,organization,repository}`\n'))
+                    '`machete.github.[remote,organization,repository]`\n'))
```

Comment:
Hmm I wanted to use the "brace expansion" syntax to evoke the [analogous mechanism in shell](https://www.gnu.org/software/bash/manual/html_node/Brace-Expansion.html)... square brackets are more typically use to signify optional stuff. Maybe leave braces as they were, just add some `noqa`?